src/components/lists/Lists.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div>
```

Comment:
Shouldn't we better locate these lists in a grid-manner layout instead of a block one? There's a huge lot of unused horizontal space left.

![Screenshot_20190719_181133](https://user-images.githubusercontent.com/9782236/61545709-c3c3b780-aa50-11e9-939d-4198ceb88c26.png)

Reply:
thanks, I'll consider this